### PR TITLE
🌐 i18n on README

### PR DESCRIPTION
**Pull request by aidakdev**
This contribution adds a README.es-ES that is translated to Spanish.
The original README has got a link to switch to the Spanish file, in this one a link to switch to the original is added.

imagen
imagen

### CODE REVIEW

File: README.es-ES.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+**AnnounceIt** es un bot de Discord diseñado para facilitar la comunicación en servidores con varios idiomas. Permite publicar mensajes con soporte para múltiples lenguajes y proporciona un botón para cambiar entre idiomas, asegurando que todos los usuarios puedan entender los anuncios importantes.
+
+<div align="center">
+<img src=".github/showcase_announceit1.gif" style="border-radius: 15px;" alt="showcase gif">
+</div>
+
+## Características
+
+- Crear un anuncio usando los modals nativos de Discord.
```
Comment (Vicente015) [inline README.es-ES.md]:
```suggestion
- Crear un anuncio usando los formularios nativos de Discord.
```

File: README.es-ES.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+- Personalizar con imágenes, miniaturas, título y pie de página en el embed.
```
Comment (Vicente015) [inline README.es-ES.md]:
```suggestion
- Personalizar con imágenes, miniaturas, título y pie de página el embed.
```

**Review by Vicente015 — CHANGES_REQUESTED** [review]
Thank you! Some words can be improved :)

**Review by Vicente015 — APPROVED** [review]
Everything fine! :)

**Comment by aidakdev** [comment]
Thank you!

**Pull request merged**